syne_tune/backend/sagemaker_backend/sagemaker_backend.py

```diff
@@ -326,3 +326,7 @@ def set_path(
     ):
         # we use the tuner-name to set the checkpoint directory
         self.tuner_name = tuner_name
+
+    def on_tuner_save(self):
+        # Re-initialize the session after `Tuner` is serialized
+        self.initialize_sagemaker_session()
```

Comment:
Could you take this change out? It seems not relevant to the import, also reinitializing the session at all saving could have undesirable effects.